tools/postman/Makefile

```diff
@@ -44,3 +45,16 @@ compare_and_send_forks: compare_forks send_forks
 .PHONY: clean
 clean:
 	rm ./openapi/*; rm ./tmp/*
+
+# Command to load local environment variables from file
+load-env:
+	@if [ -f $(ENV_FILE) ]; then \
+		echo "Loading environment variables from $(ENV_FILE)..."; \
+		for var in $$(grep -v '^#' $(ENV_FILE) | cut -d= -f1); do \
+			echo "Loaded: $$var"; \
+		done; \
+		export $$(grep -v '^#' $(ENV_FILE) | xargs); \
+	else \
+		echo "Error: $(ENV_FILE) file not found."; \
+		exit 1; \
+	fi
```

Comment:
can this be done by 

`source local.env` ?


Side note. As we discussed before, I think we have overgrown bash use cases. 
If we continued working on postman, it would be good to use some scripting language.

Reply:
> source local.env won't work in the makefile. 

Yes but having logic for credentials processing in makefile is not best as well. 
Typically make file calls single script. 

I never have strong opinions. Only flagging some common pattern. Approved PR.